tests/tests.rs

```diff
@@ -189,14 +189,54 @@ fn eq_option_err_07() {
                "failed to parse 'a' cause invalid digit found in string");
 }
 
-#[cfg(not(feature = "eq-separator"))]
+#[cfg(not(any(feature = "eq-separator", feature = "short-space-opt")))]
 #[test]
 fn no_eq_separator_01() {
     let mut args = Arguments::from_vec(to_vec(&["-w=a"]));
     let value: Result<Option<u32>, Error> = args.opt_value_from_str("-w");
     assert_eq!(value.unwrap(), None);
 }
 
+#[cfg(feature = "short-space-opt")]
+#[test]
+fn space_option_01() {
+    let mut args = Arguments::from_vec(to_vec(&["-w10"]));
+    let value: Option<u32> = args.opt_value_from_str("-w").unwrap();
+    assert_eq!(value.unwrap(), 10);
+}
+
+#[cfg(feature = "short-space-opt")]
+#[test]
+fn space_option_02() {
+    let mut args = Arguments::from_vec(to_vec(&["-w--width"]));
```

Comment:
This is very error prone... But I guess we can't do anything about it.

Reply:
I agree, but that's on the end user to not mess up.